client/components/Slide/Components/Text/Editor.jsx

```diff
@@ -7,24 +7,37 @@ import { type } from './type';
 class TextEditor extends React.Component {
     constructor(props) {
         super(props);
-        const defaultValue = convertFromHTML(props.value.html) || '';
+
+        const defaultHtml = props.value.html || '';
+        const hasWrapper = /^(<.+>.*<\/.+>)$/gm.test(defaultHtml);
+        const defaultValue = convertFromHTML(
+            hasWrapper ? props.value.html : `<p>${props.value.html}</p>`
```

Comment:
I can't quite hit this error again for some reason, but I found on legacy content this line was throwing an error because `props.value` didn't exist.

Reply:
Fixed!